tests/GenerateDocumentationTest.php

```diff
@@ -242,9 +242,9 @@ public function generated_postman_collection_file_is_correct()
         config(['apidoc.routes.0.match.prefixes' => ['api/*']]);
         $this->artisan('apidoc:generate');
 
-        $generatedCollection = json_decode(file_get_contents(__DIR__.'/../public/docs/collection.json'));
-        $generatedCollection->info->_postman_id = '';
-        $fixtureCollection = json_decode(file_get_contents(__DIR__.'/Fixtures/collection.json'));
+        $generatedCollection = json_decode(file_get_contents(__DIR__.'/../public/docs/collection.json'), true);
+        $generatedCollection['info']['_postman_id'] = '';
```

Comment:
I can see why this is necessary, but please add a comment. Otherwise, at first, it looks like we're "hacking" one of our own tests. 😅

Reply:
Same here, copied that from the existing implementation. Added the comment everywhere now.